golang/main.go

```diff
@@ -0,0 +1,113 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"io"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"time"
+)
+
+func main() {
+	// insert URL you have provided to Benzinga
+	http.HandleFunc("/",  WebHookHandler)
+	//optional: set port to other number
+	err := http.ListenAndServe(":5001", nil)
+	if err != nil {
+		log.Fatal(err)
+	}
+	log.Printf("server start")
```

Comment:
I believe `http.ListenAndServe` is a blocking operation and so this won't print until it stops.